thesis-de/content-de.tex

```diff
@@ -2669,7 +2669,33 @@ \section{Realisierung Grafana}
 \mr%
 
 
-\section{Tests und Qualitätssicherung}
+\chapter{Tests und Qualitätssicherung}
+Der genutzte, und auch der selbst entwickelte, Code ist äußerst komplex. Es
```

Comment:
ne in Verbindung entweder das "," aber kein Und oder das und aber dann kein ",". Ein Und zusammen mit einem "," gibt es so nicht bin ich der Meinung :/